lib/handbook/addon/docs/template.hbs

```diff
@@ -1,11 +1,12 @@
-{{#docs-viewer as |viewer|}}
-    {{#viewer.nav project=model as |nav|}}
+<DocsViewer as |viewer|>
+    <viewer.nav project=model as |nav|>
         {{nav.section 'Introduction'}}
```

Comment:
We can change this to angle bracket invocation like this:
```hbs
<nav.section @label='Introduction' />
```

Reply:
Yeah, that's much better. I had peeked in the source and thought it only did positional parameters.